docs/fundamentals/as-avs.txt

```diff
@@ -0,0 +1,262 @@
+
+Autocomplete Queries
+````````````````````
+
+The {+odm-short+} provides the ``autocomplete()`` method as a query
+builder method and as an Eloquent model method. You can use the
+``autocomplete()`` method to run autocomplete searches on documents in your
+collections. This method returns only the values of the field you
+specify as the query path.
+
+To learn more about this type of Atlas Search query, see the
+:atlas:`autocomplete </atlas-search/autocomplete/>` reference in the
+Atlas documentation.
+
+.. note::
+
+   You must create an Atlas Search index with an autocomplete configuration
+   on your collection before you can perform autocomplete searches. See the
+   :ref:`laravel-as-index` section of this guide to learn more about
+   creating Search indexes.
+
+The following code performs an Atlas Search autocomplete query for the
+string ``"jak"`` on the ``title`` field:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/fundamentals/as-avs/AtlasSearchTest.php
+      :language: php
+      :dedent:
+      :start-after: start-auto-query
+      :end-before: end-auto-query
+
+   .. output::
+      :language: json
+      :visible: false
+
+      [
+         "Dreaming of Jakarta",
+         "Jakob the Liar",
+         "Emily Calling Jake"
+      ]
+
+You can also pass the following parameters to the ``autocomplete()``
+method to customize the query:
+
+- ``fuzzy``: ``bool`` or ``array`` (default: ``false``)
+- ``tokenOrder``: ``string`` (default: ``'any'``)
+
+To learn more about these parameters, see the :atlas:`Options
+</atlas-search/autocomplete/#options>` section of the
+``autocomplete`` operator reference in the Atlas documentation.
+
+Atlas Vector Search
+-------------------
+
+In this section, you can learn how to create Atlas Vector Search indexes
+and perform searches in the {+odm-short+}. The Atlas Vector Search API
+internally uses the ``$vectorSearch`` aggregation operator to perform queries.
+To learn more about this operator, see the :atlas:`$vectorSearch
+</atlas-vector-search/vector-search-stage/#mongodb-pipeline-pipe.-vectorSearch>`
+reference in the Atlas documentation.
+
+.. _laravel-avs-index:
+
+Create an Atlas Vector Search Index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. TODO in DOCSP-46230
+
+Perform Queries
+~~~~~~~~~~~~~~~
+
+In this section, you can learn how to use the Atlas Vector Search API in
+the {+odm-short+}. The {+odm-short+} provides the ``vectorSearch()``
+method as a query builder method and as an Eloquent model method. You
+can use the ``vectorSearch()`` method to run Atlas Vector Search queries
+on documents in your collections.
+
+You must pass the following parameters to the ``vectorSearch()`` method:
+
+- ``index``: Name of the vector search index (type: ``string``)
+- ``path``: Field that stores vector embeddings (type: ``array`` or ``string``)
+- ``queryVector``: Vector representation of your query (type: ``array``)
+- ``limit``: Number of results to return (type: ``int``)
```

Comment:
S: The parentheses look a bit weird on the page to me. Consider putting this information in a table or moving the type information by the parameter name.
```suggestion
- ``index`` (type: ``string``) : Name of the vector search index
- ``path`` (type: ``array`` or ``string``): Field that stores vector embeddings 
- ``queryVector`` (type: ``array``): Vector representation of your query 
- ``limit`` (type: ``int``): Number of results to return
```